### PR TITLE
Updates politicians show page to custom path, connected to #4

### PR DESCRIPTION
**Pull request by drod1000**

### CODE REVIEW

**Comment by Dpalazzari** [comment]
@drod1000 @Laszlo-JFLMTCO  Ready to merge!

**Comment by njgheorghita** [comment]
looks good!

File: spec/features/user_sees_a_politicians_outings_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.feature "visitor visits '/politicians_name'" do
+  scenario "they can view all outings related to politician" do
+
+    politician = create(:politician)
+    outing_1, outing_2 = create_list(:outing, 2, politician: politician)
+
```
Comment (Laszlo-JFLMTCO) [inline spec/features/user_sees_a_politicians_outings_spec.rb:8]: @drod1000 @Dpalazzari @njgheorghita  At our next update, can we expand on the test case and bring back the removed `politician_2` and its `outings` and have confirmation that those outings are not listed on the page?

Comment (njgheorghita) [reply spec/features/user_sees_a_politicians_outings_spec.rb:8]: yeah that sounds like a great idea, robust tests ftw

Comment (Dpalazzari) [reply spec/features/user_sees_a_politicians_outings_spec.rb:8]: Yeah sure I don't see why not. Good testing is never a bad thing.

**Comment by Laszlo-JFLMTCO** [comment]
@drod1000 Good job with the implementation. I added a comment regarding expanding the test case with confirmation that other politician's outings are not listed on the page. Otherwise I am OK to merge. FYI: @Dpalazzari @njgheorghita

**Pull request merged**